content/en/security/application_security/guide/manage_account_theft_appsec.md

```diff
@@ -0,0 +1,669 @@
+2. Look for traces tagged with successful and failed login activity from your login service. For example, in **Search for**, you might have all.  
+3. Open a trace.  
+4. In the trace details, is the **Security** tab, review **Business Logic Event**.  
+5. Check if the event has a false user.
+
+<!-- ![][image5] -->
+
+In the event of a **false** user (`usr.exists:false`), look for the following issues:
+
+- A single event: if the trace contains multiple login events, such as both successes and failures, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).  
+- If the event does not contain the mandatory metadata, it might appear as a user attribution section. The mandatory metadata is `usr.login` and `usr.exists` in the case of login failure, and `usr.id` in the case of login success. In this case, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).
+
+**If the instrumentation is correct, go to [Phase 2: Preparing for Account Takeover campaigns](#phase-2:-preparing-for-account-takeover-campaigns).**
+
+### Step 1.5: Manually instrumenting your services
+
+ASM collects login information and metadata using an SDK embedded in the Datadog libraries. Instrumentation is performed by calling the SDK when a user login is successful/fails and by providing the SDK with the metadata of the login. The SDK attaches the login and the metadata to the trace and sends it to Datadog where it is retained.
+
+**For an alternative to modifying the code of the service**, go to [Step 1.6: Remote instrumentation of your services](#step-1.6:-remote-instrumentation-of-your-services).
+
+To manually instrument your services, do the following:
+
+1. If auto-instrumentation is providing incorrect data (multiple events in a single trace), see [Disable auto-instrumentation][9].
+   For detailed instrumentation instructions for each language, go to [Adding business logic information (login success, login failure, any business logic) to traces][10]. 
+2. Add the following metadata:
+   * `usr.login`: **Mandatory for login success and failure**. This field contains the *name* used to log into the account. The name might be an email address, a phone number, a username, or something else. The purpose of this field is to identify targeted accounts even if they don't exist in your systems because a user might be able to change those accounts.
+   * `usr.exists`: **Recommended for login failures**. This field is required for some default detections. The field helps to lower the priority of attempts targeted at accounts that don't exist in your systems.  
+   * `usr.id`: **Recommended for login success and failure (if available)**. This field contains a unique identifier for the account. User blocking is based on this value. This field also helps with investigation. If no identifier is available (because the account doesn't exist), you don't need to populate this field.
```

Comment:
```suggestion
   * `usr.id`: **Mandatory for login success, recommended for login failures (if the user exists)**. This field contains a unique identifier for the account. User blocking is based on this value. This field also helps with investigation to extract the post-compromise activity. If no identifier is available (because the account doesn't exist), you don't need to populate this field.
```